about.html

```diff
@@ -18,6 +18,13 @@
 	</head>
 
 	<body>
+		<div class="social_nav">
+			<div class="ic_fb"></div>
```

Comment:
- [ ] The previous reviewer mentioned that the mobile menu at the top is barely readable because of the background image, and requested to either change it/remove it, or make the menu occupy a bigger portion of the page and give it another background. But, I can see that it is still barely readable. Please consider making the suggested changes. 

![image](https://user-images.githubusercontent.com/77038610/152324700-1962de73-0bf5-408d-8314-6f86506edf15.png)